src/core/genderRatioConverter.js

```diff
@@ -1,31 +1,15 @@
 export const convertGenderRatioFromDecimal = (decimalValue) => {
-  if (decimalValue <= 0) {
+  if (decimalValue === 0) {
     return { male: 100.0, female: 0.0 };
   }
-
-  if (decimalValue <= 31) {
-    return { male: 88.14, female: 11.86 };
+  if (decimalValue < 254) {
+    return { 
+      male: 100-((100*(decimalValue-1))/253),
```

Comment:
We should probably add .toFixed here or something, high likelihood JS returns a buggy decimal at some point in numerical format.